cov.el

```diff
@@ -226,23 +226,37 @@ code's execution frequency"
 (defun cov--help (n percentage)
   (format "cov: executed %d times (~%.2f%% of highest)" n (* percentage 100)))
 
-(defun cov--set-overlay (line max)
+(defun cov--set-overlay (line max displacement)
   (let* ((times-executed (nth 1 line))
          (percentage (/ times-executed (float max)))
-         (overlay (cov-make-overlay
-                   (cl-first line)
+         (overlay (cov--make-overlay
```

Comment:
It doesn't look like the `cov-make-overlay` was changed in this commit. I agree that it should be private, though.